contents/docs/billing/common-questions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Common questions about billing
+---
+
+### Can I have a trial?
+
+PostHog is usage-based and you can use it for free for as long as you want (up to 1M events and 15k session replays per month), with no trial needed.
+
+If you have special circumstances where this doesn't work for you, contact our [sales team](/contact-sales). 
```

Comment:
```suggestion
If you have special circumstances where this doesn't work for you, contact our [customer success team](/contact-sales). 
```

Reply:
Actually I think for new customers we refer to them as sales (emails go to sales@posthog.com for instance) and then once they are customers we refer to it as CS.